### DISCUSSION TITLE
amamba/user-guide/pipeline/install-jenkins/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1748] [Announcements]
# amamba/user-guide/pipeline/install-jenkins/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/amamba/user-guide/pipeline/install-jenkins/

**Comment by liyuerich** [comment]
安装 Jenkins 之前需要确保将要 Jenkins 的集群中存在默认的存储类。

这句话不太通顺： 确保将要 Jenkins 的集群？ 是不是应该为 “确保将要 安装Jenkins 的集群”

**Comment by liyuerich** [comment]
jenkins安装表单中现在有 Agent，需要说明

**Comment by muma378** [comment]
> 如果集成的 Jenkins 实例部署在非 kpanda-global-cluster 集群时，会导致应用工作台无法更新 Jenkins 实例的配置文件（后续版本时会优化集成 Jenkins 实例需要填写所在集群、命名空间），从而引发出下述两个问题：

1. 部署在非 kpanda-global-cluster 集群下的amamba-system 命名空间式。有ns的要求；
2. 后续版本时会优化。但是需要说明下这个本身也是非必填的呢，即使后续加上了这两个字段，也还是会有问题。改成在 **或者集成时Jenkins实例时没有提供集群和命名空间**
